### PR TITLE
Modify models generated by sequelize-auto.

### PR DESCRIPTION
**Pull request by sharanry**
I have changed a few things. An extra column called 'id' has been added to csa and institure_holidays. i didn't understand why. Is it just to create primary key?

### CODE REVIEW

File: src/data/models/institure_holidays.js [hunk 1]
```diff
@@ -4,15 +4,17 @@ import DataTypes from 'sequelize';
 import Model from '../sequelize';
 
 const Institure_Holidays = Model.define('institure_holidays', {
-  id: {
-    type: DataTypes.INTEGER(11).UNSIGNED,
-    allowNull: false,
-    autoIncrement: true,
-    primaryKey: true,
-  },
+  // required?
```
Comment (mukkachaitanya) [inline src/data/models/institure_holidays.js]: This is being used as serial number (or row number) for the holidays.

Comment (mukkachaitanya) [inline src/data/models/institure_holidays.js]: Also clearly I don't see its use.

File: src/data/models/institure_holidays.js [hunk 2]
```diff
@@ -4,15 +4,17 @@ import DataTypes from 'sequelize';
...
+  // id: {
+  //   type: DataTypes.INTEGER(11).UNSIGNED,
+  //   allowNull: false,
+  //   autoIncrement: true,
+  //   primaryKey: true,
+  // },
   date: {
```
Comment (mukkachaitanya) [inline src/data/models/institure_holidays.js]: Might as well change it to [DATE](http://docs.sequelizejs.com/variable/index.html#static-variable-DataTypes) DataType.

File: src/data/models/institure_holidays.js (hunk 1 above)
Comment (sharanry) [inline src/data/models/institure_holidays.js]: So what do you suggest? should I keep it?

File: src/data/models/institure_holidays.js (hunk 2 above)
Comment (sharanry) [inline src/data/models/institure_holidays.js]: But the sql dump has it in BIGINT, won't it be incompatible?

**Comment by mukkachaitanya** [comment]
Another small change I would suggest is to add `timestamps: false` in the sequelize global config in `sequelize.js`.
This will disable the creation of `createdAt` and `updatedAt` timestamps columns.

File: src/data/models/institure_holidays.js (hunk 1 above)
Comment (mukkachaitanya) [inline src/data/models/institure_holidays.js]: The problem with sequelize data models is they expect a column with primary key, typically a column named `id`. 
If there exists no column which is primary key, sequelize created its own column named `id`. ( [Source](https://stackoverflow.com/a/29234106) )

**Comment by sharanry** [comment]
I had to make a new pull request #33

**Pull request closed without merging**